src/utils/tests/env.test.ts

```diff
@@ -184,3 +185,23 @@ describe('getReferralHashtags', () => {
         );
     });
 });
+
+describe('getSubgraphUrl', () => {
+    it('should return the value of the environment variable', () => {
+        process.env.NEXT_PUBLIC_SUBGRAPH_URL_314 = 'test';
+        const subgraphUrl = getSubgraphUrl(314);
+        expect(subgraphUrl).toBe('test');
+        expect(typeof subgraphUrl).toBe('string');
+    });
+
+    it('should return empty if variable is not set', () => {
```

Comment:
An empty string also means the variable is set.

Reply:
I fixed the test.